auxx/src/Command/Proc.hs

```diff
@@ -275,6 +277,18 @@ createCommandProcs hasAuxxMode printAction mSendActions = rights . fix $ \comman
                \ using secret key #i"
     },
 
+    let name = "apply-blockchain-dump" in
+    needsAuxxMode name >>= \Dict ->
+    return CommandProc
+    { cpName = name
+    , cpArgumentPrepare = identity
+    , cpArgumentConsumer = getArg tyFilePath "file/directory"
```

Comment:
that's a bad name for a keyword argument, I'm not sure a slash will be even parsed appropriately